CRM/Contribute/BAO/Contribution.php

```diff
@@ -374,7 +374,8 @@ public static function calculateMissingAmountParams(&$params, $contributionID) {
         $params['fee_amount'] = 0;
       }
     }
-    if (!isset($params['net_amount'])) {
+    // recalculate net amount if tax amount is set
```

Comment:
you IF condition will be true if (  tax amount is set **OR** net amount is not set ) .. but this inline comment only talk about tax amount ... so can you please amend this comment to say that . something like :

`// recalculate net amount if it is not set or if tax amount is set`